src/com/xilinx/rapidwright/edif/EDIFLibrary.java

```diff
@@ -214,12 +252,17 @@ public void uniqueifyCellsWithPrefix(String name){
 	 * the list.  This is a requirement when exporting the EDIF to a file.
 	 * @return The ordered list.
 	 */
```

Comment:
Spacing looks a bit funny. Plus document `@param stable` and why you would want it on/off.